### ISSUE TITLE
Allow * on left side of input for SimpleCommonRulesRewriter

### ISSUE THREAD

**Issue by Alien667** [querqy/querqy#160]
Problem:
Currently * rules like

`Herren* => Herren $1`

only work for * on the right side of the input.

In a lot of cases this could be also helpful for left hand sided matches, e.g. for Samsung TV alternatives

`*TU7199 => $1TU7079`

allowing `55TU7199` to find `55TU7079`
`TU7199` to find `TU7079`
and `65TU7199` to find `65TU7079`

**Comment by renekrie** [comment]
Yeah, that's been sitting at the back of our minds for some time...

Could you use a replace rewriter https://docs.querqy.org/querqy/rewriters/replace.html as a workaround for now?

**Comment by Alien667** [comment]
That's what we did in fact. But that only works, if the substitution is OK and the original terms do not need to be searched. In our example case, the other TV will never be sold here, so it's fine :) But generally this would be terrific to have as an additional option.

**Comment by renekrie** [comment]
Implemented in #1127. Also see issue #1126 (which duplicates this one)
